assets/stylesheets/sass/components/_select.scss

```diff
@@ -9,11 +9,16 @@
   cursor: pointer;
   display: block;
   font-family: $base-font-family;
-  font-size: 1em;
+  font-size: 0.9em;
   margin: 0.2em 0;
-  padding: 1em 0.7em;
+  padding: 0.7em 0.7em;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0.7em` instead of `0.7em 0.7em`